source/_integrations/fjaraskupan.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: Fjäråskupan
+description: Instructions on how to configure fjäråskupan integration.
+ha_category:
+  - Binary Sensor
+  - Fan
+  - Light
+ha_release: 2021.7.2
```

Comment:
```suggestion
ha_release: 2021.8
```
Or later, but we never introduce new integrations in patch releases 😉

Reply:
tTrue that. not too likely it will hit .8 anyway :). But let's keep it at that for now.